docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -265,6 +266,7 @@ N/A
 
 **Usage notes:**
 
+* ``fromtags()`` does consume nodes by design.
```

Comment:
Neutral on whether we need this - assumption to date has been that an operator consumes nodes unless otherwise specified.

Reply:
removed